web/src/engine/websites/InManga.ts

```diff
@@ -0,0 +1,96 @@
+import { Tags } from '../Tags';
+import icon from './InManga.webp';
+import { Chapter, DecoratableMangaScraper, Manga, Page, type MangaPlugin } from '../providers/MangaPlugin';
+import { FetchCSS, FetchJSON, FetchRequest } from '../FetchProvider';
+import * as Common from './decorators/Common';
+
+type APIChapters = {
+    data: string;
+}
+
+type APIChapter = {
+   result : {
+        Identification: string,
+        FriendlyChapterNumber: string
+    }[]
+}
+
+@Common.ImageDirect()
+
+export default class extends DecoratableMangaScraper {
+
+    public constructor() {
+        super('inmanga', `InManga`, 'https://inmanga.com', Tags.Language.Spanish, Tags.Media.Manga, Tags.Source.Aggregator);
+    }
+
+    public override get Icon() {
+        return icon;
+    }
+
+    private _createPostRequest(uri: string, form: string) {
+        const request = new FetchRequest(uri, { body: form, method: 'POST', referrer: this.URI.href });
+        request.headers.set('content-type', 'application/x-www-form-urlencoded; charset=UTF-8');
+        request.headers.set('X-Requested-With', 'XMLHttpRequest');
+        return request;
+    }
+
+    public override ValidateMangaURL(url: string): boolean {
+        return /https?:\/\/inmanga\.com\/ver\//.test(url);
+    }
+
+    public override async FetchManga(provider: MangaPlugin, url: string): Promise<Manga>{
+        const id = url.split('/').pop();
+        const request = new FetchRequest(url);
+        const title = await FetchCSS<HTMLElement>(request, 'div.panel-heading h1');

```

Comment:
May use `HTMLHeadingElement` which corresponds to `h1` tag instead of `HTMLElement`